Sources/RNLibreToolsiOS13/LibreToolsResources/LibrePro.swift

```diff
@@ -192,8 +192,32 @@ class LibrePro: Sensor {
          */
     }
 
+    struct SensorInfo {
```

Comment:
We can reuse existing one, we just need to define it outside of the Sensor class